src/pint/event_toas.py

```diff
@@ -183,6 +194,13 @@ def load_fits_TOAs(
     if weights is not None:
         new_kwargs["weights"] = weights
 
+    # mask out times/columns outside of mjd range
+    mjds_float = np.asarray([r[0] + r[1] for r in mjds])
+    idx = np.logical_and((mjds_float > minmjd), (mjds_float < maxmjd))
```

Comment:
`&` would be clearer here, as well as putting them in order ((a<b) & (b<c)); sadly a<b<c doesn't work for arrays.

Reply:
Agree.  I left it as originally written but with another vote I'll change!

Reply:
I wrote it originally because I didn't think & did an element-wise and.  If there is a better way than `logical_and`, that is fine with me.

Reply:
Yeah, most of those operators are overloaded to numpy functions, so ~, &, and | all work as expected.

You know what blew my mind the other day?  @ implements matrix multiplication on arrays (as opposed to casting to np.matrix and using *).  I had no idea...

Reply:
That was a many-year battle, the numpy folks trying to get that into mainline python. And good God it came in in 3.5, so we can use it in PINT.